src/writer/blocks/shared_blueprint.py

```
@@ -0,0 +1,158 @@
+            return_value = self.runner.run_blueprint_from_components(
+                components,
+                expanded_execution_environment,
+                title=f"Shared blueprint execution ({blueprint_name})"
+            )
+            
+            self.result = return_value
+            self.outcome = "success"
+        except BaseException as e:
+            self.outcome = "error"
+            self.message = f"<pre>{traceback.format_exc()}</pre>"
+            raise e
```

Ruff (0.14.6)

Use `raise` without specifying exception name

Remove exception name

(TRY201)